### PR TITLE
Added new interesting projects

### PR DESCRIPTION
**Pull request by rpliva**

### CODE REVIEW

**Review by augi — APPROVED** [review]

**Comment by petr-k** [comment]
Before merging this, make sure that all newly added repositories have a good description.

**Comment by rpliva** [comment]
All repositories have good description. Only `scala-hashes` has quite short description but the project is small and the description is good enough for that.

**Comment by rpliva** [comment]
I cannot merge it because I don't have write access to this repository

**Pull request merged**
